dockstore-common/src/main/java/io/dockstore/common/yaml/DockstoreYamlHelper.java

```diff
@@ -373,40 +382,82 @@ private static void addNewPropertyClassToQueue(String dockstoreYamlPackageName,
         }
     }
 
-    private static <T> void validate(final T validatee) throws DockstoreYamlException {
+    public static <T> void validate(final T validatee) throws DockstoreYamlException {
+        validate(validatee, true);
+    }
+
+    public static <T> void validate(final T validatee, final boolean validateEntries) throws DockstoreYamlException {
+        validate(validatee, validateEntries, ".dockstore.yml");
```

Comment:
Maybe worth creating a constant for `.dockstore.yml` because I've seen it a few times in this file